src/git-auth-helper.ts

```diff
@@ -109,14 +114,40 @@ class GitAuthHelper {
     } else {
       await fs.promises.writeFile(newGitConfigPath, '')
     }
-
     try {
       // Override HOME
       core.info(
         `Temporarily overriding HOME='${this.temporaryHomePath}' before making global git config changes`
       )
       this.git.setEnvironmentVariable('HOME', this.temporaryHomePath)
+    } catch (err) {
```

Comment:
why do we need a catch around `code.info` and `this.git.setEnvironmentVariable`?

Reply:
removed!